### PR TITLE
Scaffold new 20-0247 form-app

### PR DESCRIPTION
**Pull request by tlei123**
## Summary

- Create Presidential Memorial Certificate (PMC) Request (40-0247) form directory & skeleton.
- Team: Platform VA Product Forms
- Not using Flipper currently

## Related issue(s)/PR(s)

- _Link to ticket created in va.gov-team-forms repo_
department-of-veterans-affairs/va.gov-team-forms#451
- Companion [content-build PR](https://github.com/department-of-veterans-affairs/content-build/pull/1664).

## Testing done

N/A &mdash; this is currently just a skeleton.

## What areas of the site does it impact?

N/A &mdash; this is a new app.

### CODE REVIEW

File: src/applications/simple-forms/40-0247/containers/ConfirmationPage.jsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { format, isValid } from 'date-fns';
+import { connect } from 'react-redux';
+
+import scrollToTop from 'platform/utilities/ui/scrollToTop';
+import { focusElement } from 'platform/utilities/ui';
+
+export class ConfirmationPage extends React.Component {
```
Comment (rhasselle-oddball) [inline src/applications/simple-forms/40-0247/containers/ConfirmationPage.jsx:9]: do we want to use the `/shared/components/ConfirmationPageView` for this?

File: src/applications/simple-forms/40-0247/containers/IntroductionPage.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+import { focusElement } from 'platform/utilities/ui';
+import OMBInfo from '@department-of-veterans-affairs/component-library/OMBInfo';
+import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
+import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
+
+class IntroductionPage extends React.Component {
```
Comment (rhasselle-oddball) [inline src/applications/simple-forms/40-0247/containers/IntroductionPage.jsx:8]: same question about `/shared/components/IntroductionPageView`. I know that yeoman gives you these pages instead but maybe we can switch to using the shared components in the future

**Review by rhasselle-oddball — APPROVED** [review]

File: src/applications/simple-forms/40-0247/containers/ConfirmationPage.jsx (hunk 1 above)
Comment (tlei123) [reply src/applications/simple-forms/40-0247/containers/ConfirmationPage.jsx:9]: I'll look into this, and open another PR if that works out better.  Thanks!

File: src/applications/simple-forms/40-0247/containers/IntroductionPage.jsx (hunk 2 above)
Comment (tlei123) [reply src/applications/simple-forms/40-0247/containers/IntroductionPage.jsx:8]: yes, will try that out during Build-frontend step.  Thanks!

**Pull request merged**